hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/fair/converter/TestFSQueueConverter.java

```diff
@@ -150,19 +152,19 @@ public void testConvertQueueHierarchy() {
 
     // root children
     assertEquals("root children", "admins,users,misc,default",
-        csConfig.get(PREFIX + "root.queues"));
+        capacitySchedulerConfig.get(PREFIX + "root.queues"));
```

Comment:
There is a helper method called getQueues, which returns the list of queues that could be used here.